src/components/KTreeList/KTreeItem.vue

```diff
@@ -3,14 +3,25 @@
     class="tree-item"
     :class="{
       'not-draggable': disabled,
-      'selected': item.selected
+      'selected': item.selected,
+      'expanded': collapsable && isExpanded,
+      'collapsed': collapsable && !isExpanded
     }"
     :data-testid="`tree-item-${item.id}`"
     data-tree-item-trigger="true"
     draggable="false"
     type="button"
     @click.prevent="handleClick"
   >
+    <component
+      :is="expandedIcon"
+      v-if="collapsable"
+      class="tree-item-expanded-icon"
+      data-testid="tree-item-expanded-icon"
+      role="button"
```

Comment:
We should add a dynamic `aria-label` for "Collapse" and "Expand," as well as `aria-controls` and `aria-expanded` (you'll need to generate a unique ID with `useId()`